### DISCUSSION TITLE
Grammar::parameterize(): Argument #1 ($values) must be of type array

### DISCUSSION THREAD

**Discussion by ManuelRomanato** [filamentphp/filament#14976] [Help]
### Package

Panel builder

### Package Version

v. 3.0.0

### How can we help you?

Hello,

I encountered an issue while saving a new category in my application. Here's the complete error log:

`Illuminate\Database\Grammar::parameterize(): Argument #1 ($values) must be of type array, string given, called in /Users/poseidone/projects/oa-cms-laravel-filament/vendor/laravel/framework/src/Illuminate/Database/Query/Grammars/Grammar.php on line 1180`

**Context:**
I have a custom resource that extends the CategoryResource from Filament.
My custom resource uses a custom model, which extends the original Filament Category model. This customization is necessary to make the database fields translatable.

**Problem:**
When I attempt to save a new category, the error above is triggered. However, when I debug the code, I see that the value being passed appears to be an array, which should be correct.

P.S.
I'm using
-  Filament v.3
- [FBlog](https://filamentphp.com/plugins/firefly-blog)
- [Translatable Inline](https://filamentphp.com/plugins/mvenghaus-translatable-inline#configuration)

Does anyone have insights into what might be causing this issue or how to resolve it?

Thank you for your support!
<img width="1461" alt="Screenshot 2024-12-02 alle 16 03 10" src="https://github.com/user-attachments/assets/56278096-85ce-49c2-bf87-0cb91083b084">
<img width="741" alt="Screenshot 2024-12-02 alle 16 08 26" src="https://github.com/user-attachments/assets/f738178f-10ae-43e3-b090-087d369232ca">

**Comment by leandrocfe** [comment]
Please provide a minimal repo on github to reproduce this issue
